awpy/plot/utils.py

```diff
@@ -115,6 +116,69 @@ def is_position_on_lower_level(
         bool: True if the position on the lower level, False otherwise.
     """
     metadata = MAP_DATA[map_name]
-    if position[2] <= metadata["lower_level_max"]:
+    if position[2] <= metadata["lower_level_max_units"]:
         return True
     return False
+
+
+# Old function names:
+def _renaming_warning(old: str, new: str):
+    return f"""Deprecation warning: Function {old} has been renamed to {new}.
+    Please update your code to avoid future deprecation.
+    """
+
+
+def position_transform_axis(
+    map_name: str, position: float, axis: Literal["x", "y"]
+) -> float:
+    """Calls `game_to_pixel_axis` and sends warning.
+    
+    This is the old name of function `game_to_pixel_axis`. Please update
+    your code to avoid future deprecation.
+    """
+    warnings.warn(
```

Comment:
Great idea to add the deprecation warnings. I think we can drop the `_renaming_warning()` however, and just include the specific strings in the deprecation messages!